app/views/layouts/_header.html.erb

```diff
@@ -0,0 +1,12 @@
+<header class="navbar navbar-fixed-top navbar-inverse">
+  <div class="container">
+    <%= link_to "sample app", '#', id: "logo" %>
+    <nav>
+      <ul class="nav navbar-nav navbar-right">
+        <li><%= link_to "Home", home_path %></li>
+        <li><%= link_to "Help", help_url %></li>
+        <li><%= link_to "Log in", "/home" %></li>
```

Comment:
chỗ này chưa có đường dẫn phù hợp nên tạm thời để dấu `#` chứ ko dẫn về trang home như này em nhé:

`<li><%= link_to "Log in", "#" %></li>`